modules/client/test/integration/multisig-client/methods.test.ts

```diff
@@ -124,12 +124,11 @@ describe("Client Multisig", () => {
           expect(step.txHash).toMatch(/^0x[A-Fa-f0-9]{64}$/i);
           break;
         case ProposalCreationSteps.DONE:
-          expect(typeof step.proposalId).toBe("number");
+          expect(typeof step.proposalId).toBe("string");
+          expect(step.proposalId).toMatch(
+            /^0x[A-Fa-f0-9]{40}_0x[A-Fa-f0-9]{1,64}$/,
+          );
           return step.proposalId;
-          // TODO
```

Comment:
I know this is not ideal, but could we leave this comment here, to make things easier to find when we finally merge the metadata validation PR? 🙏

Reply:
This todo is done so I'd rather remove it and have clean code